testutils/utils.go

```diff
@@ -23,12 +23,16 @@ func SampleMeta() data.FileMeta {
 }
 
 func GetSqliteDB() *sql.DB {
+	os.Mkdir("/tmp/sqlite", 0777)
```

Comment:
I figured we might as well rely on the umask to determine the permissions. But you're right; there's no good reason to use 0777. I'll change this.